nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/admin/HealthCheckConfigurer.java

```diff
@@ -85,9 +85,12 @@ and Distribution License("CDDL") (collectively, the "License").  You
     @Param(name = "target", optional = true, defaultValue = "server-config")
     protected String target;
 
-    @Param(name = "enabled", optional = false)
+    @Param(name = "enabled")
     private Boolean enabled;
 
+    @Param(name = "notifierEnabled", optional = true, defaultValue = "false")
```

Comment:
We've just created a few PRs to remove default values from parameters when they're not necessary, as it essentially makes the parameter required.

If a parameter is optional, but also has a default value, it means that you could configure the healthcheck to have the notifier enabled in one command, but if you then run the command again without this parameter specified it will overwrite the current configuration and disable it again.